### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by stephen-rowan** [Council-of-DAOs/Council-Hub#5] [General]
## Introductions

This is where participants in the Council of DAOs - Council Hub can provide details of themselves and their projects. 

We would welcome a **Who I am**, your **Timezone** (the times you are active or available) and your **Projects**  (web links, Ideascale proposals).

## [Stephen Whitenstall (stephen-rowan)](https://github.com/Council-of-DAOs/Council-Hub/discussions/5#discussioncomment-858613) - (Adagov.org, NFT-DAO, Council-Hub GitHub maintainer, QA-DAO, tba )

**Comment by phillewis** [comment]
Well someone has to be first right? :)

Hi, I'm Phil Lewis.  I live in Perth, Australia (UTC+8).  I am usually active/available between 01:00 and 15:00 UTC (subject to work/family commitments).  I think this is a great initiative by Stephen and I look forward to contributing.

I have been active in the Cardano community since 2018 and currently have a few different projects on the go.

### Empowa
**Description:** A platform to enable African communities to improve living conditions and create wealth.
**Website:** https://empowa.io
**Catalyst Proposal:** https://cardano.ideascale.com/a/dtd/Introducing-Empowa/352666-48088 (funding for Phase 0 only)

### NFT DAO
**Description:** My focus in NFT DAO is around the use of NFT standards for the framework being developed and I have proposed we actively engage with groups starting to form cross chain industry standards, such as the Interwork Alliance and InterNFT
**Website:** https://nft-dao.org
**Catalyst Proposal:**  https://cardano.ideascale.com/a/dtd/NFTDAO-Industry-Standards-1-dues/352362-48088

### DARP
**Description:** The Decentralised Address Resolution Protocol will build on Atala PRISM's DID technology to allow easy to remember address names to be used
**Website:** https://docs.darp.tech
**Catalyst Proposal:** https://cardano.ideascale.com/a/dtd/DARP-Cardano-Address-Name-Service/341994-48088

**Comment by wlstylr** [comment]
Hello, I'm Tyler Wales.  I live in Portland, Oregon (UTC-7).  I have a background in product development, marketing, media, and entrepreneurship.   I first learned about Cardano in 2017 and have been an active participant in Project Catalyst since 2020.  

I can't commit to rigid times when I'm available, but do keep Wednesdays and Fridays as open as possible to accommodate Cardano related activities.   

I helped start NFT-DAO, and am now working on some other projects around NFTs, art, video, and defi.  I plan to share links when ready. 

I look forward to helping out with this initiative, and learning through action.

**Comment by newmindflow** [comment]
Hi, we’re **New Mindflow** (Anamaria & Rareș), multidisciplinary artists and entrepreneurs, currently based  in Transylvania, Romania (UTC+2 Standard Time, UTC+3 Daylight Savings).

We have a background in film direction & production, game development and art.

We’re film directors, film-makers by trade, but we left that scene some time ago to explore new avenues, both artistically and entrepreneurially. We set up a small game dev studio in Jogjakarta, Indonesia, we launched interactive art in the form of iPad apps, and we’ve more recently focused on visual art - illustrations, painting and photography both digital and analogue.

We got involved in Cardano’s Project Catalyst in February 2021, when we were looking for ways to build an art discovery and sustainability platform centred around NFTs on Cardano.

---

We have multiple Project Catalyst Proposals, both in Fund4 and Fund5:

**Fund4**

F4: Dapps & Integrations
**The Cardano Global Treasure Hunt** 

An initiative to attract new users to the Cardano ecosystem by harnessing the ingenuity of its global community to organise a unique, fun & rewarding event. (Since the original proposal we have pivoted to a partnership with WADA and refocused the initiative to West Africa)

https://cardano.ideascale.com/a/dtd/The-Cardano-Global-Treasure-Hunt/340173-48088


F4: Fund6 Challenge Setting
**Decentralised Storage Solutions**

A challenge that asks the question how can we create a robust, reliable, secure and affordable decentralised, high-capacity storage network built upon the Cardano blockchain?

https://cardano.ideascale.com/a/dtd/Decentralised-Storage-Solutions/340968-48088 

**Fund5**

F5: Developer ecosystem
**Devs for Awareness Event West Africa**

A proposal aimed at setting the groundwork for our collaboration with WADA to bring the Treasure Hunt initiative to West Africa by recruiting, training and equiping a team of African developers to build the tech infrastructure needed for a much broader event, developed in multiple stages.

https://cardano.ideascale.com/a/dtd/Devs-for-Awarenes-Event-West-Africa/350652-48088

F5: Fund7 challenge setting
**Decentralised Storage Solutions**

A continuation of the Fund4 proposal, as creating a decentralised storage solution would definitely be a multi-phase project.

https://cardano.ideascale.com/a/dtd/Decentralised-Storage-Solutions/352807-48088 


F5: Distributed decision making
**A Council of Catalyst DAOs**

An initiative to create a body where existing and emergent DAOs and Project Catalyst can communicate and document DAOs, and research ways to improve them. (The Catalyst proposal behind this repo - thanks Stephen for setting up the Hub initiative!)

https://cardano.ideascale.com/a/dtd/A-Council-of-Catalyst-DAOs/352206-48088#idea-tab-comments

---

**Projects**

**Trybbles**
The incredibly cute and fluffy collectible NFTs exclusively on Cardano that are more than what they seem.
(And that's all we'll share about it for now)
https://www.trybbles.com
https://twitter.com/trybbles

**Links:**
http://www.newmindflow.com
https://www.instagram.com/newmindflow
https://www.behance.net/freemindflow
https://www.imdb.com/name/nm3439147
https://www.youtube.com/newmindflow
https://500px.com/p/freemindflow

**Comment by stephen-rowan** [comment]
Hi there, I am Stephen Whitenstall (@stephen-rowan). I live in Todmorden, a small town in the West Yorkshire Moors, UK (UTC 0, Standard Time, UTC+1 Daylight Savings). My immediate background is in philosophy and I am interested in the development of distributed governance on Cardano. A more general profile can be found [here](https://stephen-rowan.github.io/) and on [linkedin](https://www.linkedin.com/in/stephen-whitenstall-166727210/).

I am a GitHub maintainer for the Adagov.org [documentation](https://github.com/adagovorg/adagov-documentation),  [NFT-DAO Governance-HOLON](https://github.com/NFT-DAO/Governance-HOLON), [NFT-DAO Standards](https://github.com/NFT-DAO/NFT-Standards) repos & creator of the [NFT-DAO GitBook](https://nft-dao-1.gitbook.io/nft-dao/). As well as other projects currently in development.

I assist with this repository [Council-of-DAOs/Council-Hub](https://github.com/Council-of-DAOs/Council-Hub) in support of the Council-of-DAOs proposal. My own Fund 5 proposal is [QA-DAO](https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal) – which aims to encourage collaboration and innovation across all open-source Catalyst Projects. I also like to participate in Project Catalyst more generally, connecting with and supporting its rich ecosystem.

**Reply by phillewis** [reply]
Omg @stephen-rowan, I just looked up the town where you live and I grew up 15 miles away from there in a village called Cullingworth.  Small world 😄

**Reply by stephen-rowan** [reply]
Cullingworth - that's Bronte country near Haworth just north of me. I am originally from Southwark, London, moved up here to escape the city !

**Reply by phillewis** [reply]
Yeah, Bronte home was only about a mile away.
Beautiful countryside to escape to.  It was a great place as a kid.  I'm hoping to get over there next year if things are back to normal, to show my kids my old stomping grounds 😆
